packages/wes_adapter/install.sh

```diff
@@ -1,5 +1,10 @@
 #! /bin/bash
 
+if ! hash python3.9; then
+    echo "python3.9 is not installed"
```

Comment:
Perhaps add a note saying "Please install Python 3.9 by running ..."

On the same note, I wonder if it makes sense to add Python 3.9 as a pre-requisite under the Development section in the README. Otherwise customers won't be able to run the install script due to this missing dependency.

Reply:
How to install python could depends the users' platform. I have added the instructions on Readme, but will omit printing such long instructions from this message.